Zadanie7.3-N/main.py

```diff
@@ -0,0 +1,60 @@
+import random
+from faker import Faker
+from businesscard import BusinessContact, BaseContact
+
+
+def generate_business_card(quantity=1):
+    business_cards = list()
+    fake = Faker('pl_PL')
+
+    for _ in range(quantity):
```

Comment:
zróbmy to jako for comprehension, czyli coś takiego:


`return [generate_fake_data() for _ in range(quantity)]`

Reply:
jest taka składnia https://www.w3schools.com/python/python_lists_comprehension.asp